### PR TITLE
chore(tooling): Move tests circleci

### PR DESCRIPTION
**Pull request by bzang**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit d9ed18cb5801b92dfdc86b74729a0cdacebaced2

https://deploy-preview-680--hopeful-hamilton-4090b9.netlify.com

File: Jenkinsfile [hunk 1]
```diff
@@ -78,9 +78,9 @@ ansiColor('xterm') {
             ]) {
               sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' >> .npmrc'
               sh '''#!/bin/bash -ex
```
Comment (adamweeks) [inline Jenkinsfile]: We going to remove the `-ex`?

**Comment by bzang** [comment]
Ok, this piece is ready to merge. I'm setting up the staging branch and settings in GH and circle today. Next steps:

- [ ] Update jenkins job to poll GH branch for updates

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+  install:
+    docker:
+      - image: circleci/node:8.9.4
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - js-dependencies-{{ checksum "package-lock.json" }}
+          - js-dependencies-
+      - run: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} >> .npmrc
```
Comment (adamweeks) [inline .circleci/config.yml]: Did we get a read-only NPM token to use?

File: Jenkinsfile [hunk 3]
```diff
@@ -77,81 +77,71 @@ ansiColor('xterm') {
...
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/" npm run build sri widget-space
-              BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/demo/" npm run build:package widget-space-demo
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" npm run build:package widget-recents
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" npm run build sri widget-recents
-              BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/demo/" npm run build:package widget-recents-demo
-              '''
+             sh '''#!/bin/bash -e
+             source ~/.nvm/nvm.sh &> /dev/null
+             nvm use v8.9.4
+             export JOURNEY_TEST_BASE_URL=https://practical-roentgen-7d4de0.netlify.com
```
Comment (adamweeks) [inline Jenkinsfile]: Should we make this a param somewhere or env var so that we can easily change it if necessary?

File: package.json [hunk 4]
```diff
@@ -11,20 +11,23 @@
     "build:widgets": "cross-env NODE_ENV=${NODE_ENV:-production} npm run build widgets",
     "build:sri": "npm run build sri",
     "clean": "npm run clean:dist && npm run clean:transpile",
-    "clean:dist": "rimraf \"packages/**/dist/\"",
+    "clean:dist": "rimraf \"packages/**/dist/\" && rimraf \"packages/**/dist/\"",
```
Comment (adamweeks) [inline package.json]: What am I missing here? Why is this run twice?

File: scripts/deploy/index.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env babel-node
+/**
+ * Deploy code to different paces
```
Comment (adamweeks) [inline scripts/deploy/index.js]: `paces` > `places`

**Comment by bzang** [comment]
Superseded by #709 and #709

**Pull request closed without merging**